packages/components/src/Draggable/DraggableComponent.test.js

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import TestBackend from 'react-dnd-test-backend';
+import { DragDropContext as dndContext } from 'react-dnd';
+import PropTypes from 'prop-types';
+import TestUtils from 'react-dom/test-utils';
+import draggable from './DraggableComponent.js';
+
+function Element({ element }) {
+	return (
+		<div className="elem-class">
+			<span>{element.symbol}</span>
+			<span>:</span>
+			<span>{element.name}</span>
```

Comment:
Why not use a definition list here?

Reply:
i just need a simple component for the test